Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -125,11 +119,11 @@ public class BTShopperInsightsClient {
             paymentMethodsDisplayed: paymentMethodsDisplayedString
         )
     }
-    
-    /// Call this method when the Venmo button has been selected/tapped by the buyer.
-    /// This method sends analytics to help improve the Shopper Insights feature experience
-    public func sendVenmoSelectedEvent() {
-        apiClient.sendAnalyticsEvent(BTShopperInsightsAnalytics.venmoSelected)
+
+    /// Call this method when a button has been selected/tapped by the buyer.
+    /// This method sends analytics to help improve the Shopper Insights feature experience.
+    public func sendSelectedEvent(for buttonType: BTButtonType) {
+        apiClient.sendAnalyticsEvent(buttonType.rawValue)
```

Comment:
We will want to update `BTShopperInsightsAnalytics` to remove the `venmoSelected` and `payPalSelected` events and replace with a `buttonSelected` event: `shopper-insights:button-selected`. Then we will set the button type as the button type you added in addition to the event name.